src/templates-idea/solarized-chandrian.theme.json

```diff
@@ -399,35 +405,35 @@
     "RadioButtonMenuItem": {},
     "ScrollBar": {
       "background": "{{backgroundMostIntense}}",
-      "hoverThumbBorderColor": "{{backgroundIntense}}",
-      "hoverThumbColor": "{{backgroundMild}}",
-      "hoverTrackColor": "",
+      "hoverThumbBorderColor": "{{backgroundMostMild}}",
```

Comment:
I've synced the theme scrollbar with the editor scrollbar, so some of the transparency values ended up there and all the colours ended up here. This makes the scrollbar very easy to notice everywhere and with some neat hover effects as well